src/Microsoft.NetCore.Analyzers/UnitTests/Runtime/DisposeObjectsBeforeLosingScopeTests.cs

```diff
@@ -1138,6 +1138,88 @@ End Class
             GetBasicResultAt(16, 51, "new A(2)"));
         }
 
+        [Theory]
+        [InlineData(true)]
+        [InlineData(false)]
+        public void Configured_DisposeOwnershipTransfer_AtConstructorInvocation(bool disposeOwnershipTransferAtConstructor)
+        {
+            var editorConfigText = disposeOwnershipTransferAtConstructor ?
+                        $@"dotnet_code_quality.interprocedural_analysis_kind = None
+                           dotnet_code_quality.dispose_ownership_transfer_at_constructor = true" :
```

Comment:
Tagging @sharwell - the default value (false) matches the legacy FxCop behavior, but at least we now have an option to match your preferred behavior. We can consider switching the default in future if we end up seeing lot of noise due to this.